Source/DataSources/GeoJsonDataSource.js

```diff
@@ -732,6 +732,8 @@ define([
      * @param {String|Object} data A url, GeoJSON object, or TopoJSON object to be loaded.
      * @param {Object} [options] An object with the following properties:
      * @param {String} [options.sourceUri] Overrides the url to use for resolving relative links.
+     * @param {Function} [options.describe=GeoJsonDataSource.defaultDescribe] A custom function used to convert the properties into an html description.
```

Comment:
Rather than using `Function` here, jsDoc allows us to define callback signatures.  See https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/DataSources/CallbackProperty.js#L108 for an example.